source/Sashimi.Aws.Accounts.Tests/Sashimi.Aws.Accounts.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="nunit" Version="3.12.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
+  </ItemGroup>
```

Comment:
I think we also need the Teamcity adaptor for all test projects:
`<PackageReference Include="TeamCity.VSTest.TestAdapter" Version="1.0.22" />`